java/scripts/splunk-java-config

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# disable non-lambda resource providers
+export OTEL_JAVA_DISABLED_RESOURCE_PROVIDERS="${OTEL_JAVA_DISABLED_RESOURCE_PROVIDERS:-\
```

Comment:
I've never seen this before, so I had to try it out!
Looks like it's a clever trick for setting a default....so it'll use `OTEL_JAVA_DISABLED_RESOURCE_PROVIDERS` if it's set, otherwise it uses the list of providers. 😎